crates/build/src/lib.rs

```diff
@@ -53,27 +46,27 @@ fn build_component(raw: RawComponentManifest, app_dir: &Path) -> Result<()> {
                 println!("Working directory: {:?}", workdir);
             }
 
-            let res = Exec::shell(&b.command)
+            let exit_status = Exec::shell(&b.command)
                 .cwd(workdir)
-                .stdout(Redirection::Pipe)
-                .capture()
-                .with_context(|| {
-                    format!(
-                        "Cannot spawn build process '{:?}' for component {}.",
-                        &b.command, raw.id
+                .stdout(Redirection::None)
+                .stderr(Redirection::Merge)
```

Comment:
Worth documenting why this is Merge rather than None?  (I'm secretly curious about it myself - I would have thought None would let consumers capture the streams separately, but I assume it causes some problem?)

Reply:
`Using Redirection::Merge for PopenConfig::stderr requests the child’s stderr to refer to the same underlying file as the child’s stdout (which may or may not itself be redirected),`

Reply:
Sorry, I wasn't clear - I know what it does, I was curious _why_ we wanted it to do that.

Reply:
yea you're right and I think I should change it to None.

Reply:
updated